coalib/output/ConsoleInteraction.py

```diff
@@ -649,88 +649,106 @@ def show_enumeration(console_printer,
+    :param show_params:      True if the parameters and their description
+                             should be shown.
+    :param console_printer:  Object to print messages on the console.
     """
     if not bears:
         console_printer.print("No bears to show. Did you forget to install "
                               "the `coala-bears` package? Try `pip3 install "
                               "coala-bears`.")
-    elif compress:
-        bear_list = sorted(bears.keys(), key=lambda bear: bear.name)
-        for bear in bear_list:
-            console_printer.print(bear.name)
-            console_printer.print("=" * len(bear.name))
+        return
 
-            console_printer.print(bear.get_metadata().desc + "\n")
-    else:
-        for bear in sorted(bears.keys(),
-                           key=lambda bear: bear.name):
-            show_bear(console_printer,
-                      bear,
-                      bears[bear],
-                      bear.get_metadata())
+    for bear, sections in sorted(bears.items(),
+                                 key=lambda bear_tuple: bear_tuple[0].name):
+        show_bear(bear,
+                  sections,
+                  show_description,
+                  show_params,
+                  console_printer)
 
 
-def show_bears(local_bears, global_bears, compress, console_printer):
+def show_bears(local_bears,
+               global_bears,
+               show_description,
+               show_params,
+               console_printer):
     """
     Extracts all the bears from each enabled section or the sections in the
     targets and passes a dictionary to the show_bears_callback method.
 
-    :param local_bears:         Dictionary of local bears with section names
-                                as keys and bear list as values.
-    :param global_bears:        Dictionary of global bears with section
-                                names as keys and bear list as values.
-    :param compress:            If set to true, output will be compressed (just
-                                show bear names as a list)
-    :param show_bears_callback: The callback that is used to print these
-                                bears. It will get one parameter holding
-                                bears as key and the list of section names
-                                where it's used as values.
+    :param local_bears:      Dictionary of local bears with section names
+                             as keys and bear list as values.
+    :param global_bears:     Dictionary of global bears with section
+                             names as keys and bear list as values.
+    :param show_description: True if the main description of the bears should
+                             be shown.
+    :param show_params:      True if the parameters and their description
+                             should be shown.
+    :param console_printer:  The console printer to print to.
```

Comment:
Use `Object to print messages on the console.` instead as it's consistent with the doc available in the other methods?